docs/question/template.md

```diff
@@ -44,7 +44,7 @@ Text in `question.html` can be set to only display in the "question" panel by wr
 
 ### `"submission"` panel
 
-By default, feedback given by elements (`data["partial_scores"][NAME]["feedback"]`) is rendered when an element renders in the "submission" panel. Feedback given by the question (`data["feedback"][NAME]`) needs to be shown using `{{feedback.NAME}}`.
+By default, feedback given by elements (`data["partial_scores"][NAME]["feedback"]`) is rendered when an element renders in the "submission" panel. Feedback given by the question (`data["feedback"][NAME]`) needs to be shown using `{{feedback.NAME}}`. More details about providing feedback can be found on the [`server.py` page](server.md#providing-feedback).
```

Comment:
I think the use of `data["feedback"]` is not common enough to be shown in the first paragraph of this section. Most questions will use the default element-based feedback for the submission panel, the custom feedback will be used mostly by exceptional questions that require custom grading.

Reply:
Changed in ba7cf9af7dec39288b1c7ccf81ecbc26fffd7ea3.